### ISSUE TITLE
make.bat for docs won't build

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7542]
|<img alt="amattjones's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| amattjones reported|
|-|-|
|Trac ID|trac#7542|
|Type|enhancement|
|Created|2014-07-26 12:06:23Z|
|Branch|https://github.com/twisted/twisted/tree/make-bat-path-fix-7542|

The make.bat file for documentation building on windows refers to the sources directory instead of . like in the makefile.

Attachments:

* [make.bat.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/01f/01fd21ce91283dcb3984e45e0d9945ac4aa57405/26b453a3b7bfb277a74e589f69bf9c7440338569.diff) (713 bytes) - added by amattjones on 2014-07-26 12:06:55Z - Patch for make.bat

<details><summary>Searchable metadata</summary>

```
trac-id__7542 7542
type__enhancement enhancement
reporter__amattjones amattjones
priority__low low
milestone__ 
branch__branches_make_bat_path_fix_7542 branches/make-bat-path-fix-7542
branch_author__khorn khorn
status__closed closed
resolution__fixed fixed
component__core core
keywords__docs__easy docs, easy
time__1406376383780484 1406376383780484
changetime__1410528875577386 1410528875577386
version__None None
owner__khorn khorn

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)|@khorn set owner to @khorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_4"></a>|@khorn commented|
|-|-|

This looks good to merge.  I'll see whether I can manage to remember how to do that...

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_5"></a>|@khorn commented|
|-|-|

(In [43040]) Branching to make-bat-path-fix-7542.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_6"></a>|@khorn commented|
|-|-|

(In [43042]) Add a NEWS file for make.bat path fix. refs [#7542](https://github.com/twisted/twisted/issues/7542)

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_7"></a>|@khorn set status to `closed`|
|-|-|

(In [43043]) Merge make-bat-path-fix-7542: Fix path error in Sphinx make.bat file.

Author: amattjones
Reviewers: khorn
Fixes: [#7542](https://github.com/twisted/twisted/issues/7542)

This fixes a bad path that was preset in the make.bat file used for
building Sphinx docs.  The docs can now be built on Windows using this
file.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

Shouldn't there be some test coverage for this?

Or does this stuff get excluded from the standing policy (full test coverage) because building the documentation not considered very important and it's okay for it to break sometimes?
